osdep/timer.c

```diff
@@ -73,44 +72,41 @@ int64_t mp_time_ns_add(int64_t time_ns, double timeout_sec)
     return time_ns + ti;
 }
 
-#if !HAVE_WIN32_INTERNAL_PTHREADS
-static int get_realtime(struct timespec *out_ts)
+static void get_realtime(struct timespec *out_ts)
 {
 #if defined(_POSIX_TIMERS) && _POSIX_TIMERS > 0
-    return clock_gettime(CLOCK_REALTIME, out_ts);
+    clock_gettime(CLOCK_REALTIME, out_ts);
 #else
     // OSX
     struct timeval tv;
     gettimeofday(&tv, NULL);
     out_ts->tv_sec = tv.tv_sec;
     out_ts->tv_nsec = tv.tv_usec * 1000UL;
-    return 0;
 #endif
 }
-#endif
 
 struct timespec mp_time_ns_to_realtime(int64_t time_ns)
 {
-    struct timespec ts = {0};
-
-#if !HAVE_WIN32_INTERNAL_PTHREADS
```

Comment:
This will use `gettimeofday()` now, which is not really what we want, no? `gettimeofday()` is here only for ancient macOS that I didn't have heart to break compatibility with.